themes/classic/_dev/css/modules/customer.scss

```diff
@@ -1,9 +1,19 @@
-.page-authentication {
-  .no-account {
-    text-align: center;
-    font-size: 1.25rem;
-    margin: 1rem 0;
+.table-labeled {
+  th {
```

Comment:
I agree with @Shudrum, the rscss.io norm advises us to use classnames instead of tag selectors. So, you should use the same class for your `tr` and `td`.